bug-reports.md

```diff
@@ -14,6 +14,26 @@ Once you've done a bit of searching and discovered there isn't prior art for you
 * What you expected to see.
 * What you actually saw.
 
+
+Following this template will help guarantee you are submitting a full bug report:
+
+**Environment:** affected software and versions (see below)
+
+**Summary:** in narrative form, such as “While testing feature x, I encountered z.”
+
+**Steps to reproduce the bug:** (please be as detailed and specific as possible. It's a good idea to go back and follow your steps a second time to make sure another user can recreate the issue by following the steps directly without assuming any actions)
+1. this
+2. is
+3. what
+4. you
+5. do
```

Comment:
Can you turn this into a real-world example?

Also, it'd be worth noting that WP-CLI bugs are expected to reproduce on a clean WordPress install